app/assets/stylesheets/govuk_publishing_components/components/_autocomplete.scss

```diff
@@ -0,0 +1,49 @@
+@import "govuk_publishing_components/individual_component_support";
+
+.gem-c-autocomplete {
+  position: relative;
+
+  > div {
+    margin-bottom: 0;
+  }
+  
+  .gem-c-autocomplete-result-list {
```

Comment:
We use BEM for our classes, as you've noticed. This looks like an element class, so I think it should be `.gem-c-autocomplete__result-list`?

There's a few other classes as well - I'm guessing `.gem-c-autocomplete-result` should be `.gem-c-autocomplete__result` and `.border` should be `.gem-c-autocomplete__border` - or is that one a modifier class and should be `.gem-c-autocomplete--border`?

Reply:
Also `.search-icon`